content/SCALE/SCALETutorials/Apps/StableApps/SyncthingStable.md

```diff
@@ -0,0 +1,187 @@
+
+Select the timezone where the TrueNAS server is located from the **Timezone** dropdown list.
+
+Accept the default user and group ID settings (**568**).
+If you created a user for this app, change these settings to the UID/GID for that new user.
+
+Configure the storage settings.
+You can allow Syncthing to create the configuration storage volume, but we recommend setting **Type** to **Host Path (Path that already exists on the system)**, and then enter or browse to the location of the **home** dataset to populate the **Host Path** field for the **Syncthing Config Storage** settings.
+
+{{< trueimage src="/images/SCALE/Apps/InstallSyncthingStorageConfigHomeHostPath.png" alt="Syncthing Home Storage Settings" id="Syncthing Home Storage Settings" >}}
+
+Next, Click **add** to the right of **Additional Storage** to add the storage configuration settings for the data volume.
+
+{{< trueimage src="/images/SCALE/Apps/InstallSyncthingStorageConfigData1HostPath.png" alt="Syncthing Data1 Storage Settings" id="Syncthing Data1 Storage Settings" >}}
+
+Enter or browse to the dataset created to populate the **Host Path** field (for example, */mnt/tank/syncthing/config*), then enter or browse to the location of the **data1** dataset to populate the **Host Path** field under the **Mount Path** field.
+
+To add another dataset path inside the container, see [**Storage Settings**](#storage-settings) below for more information.
+Set **Type** to **Host Path (Path that already exists on the system)**, enter **/data1** in **Mount Path**, and then either enter or browse to the path to the **data1** dataset to populate the **Host Path** field.
+
+Accept the default port numbers in **Networking**. 
+See [Network Settings](#networking-settings) below for more information on network settings. 
+Before changing the default port number, see [Default Ports](https://www.truenas.com/docs/references/defaultports/) for a list of assigned port numbers.
+When selected, **Host Network** binds to the default host settings programmed for Syncthing.
+When enabled the app uses the host network for Syncthing.
+The TCP and UDP ports listen on port **20978** and **20979**.
+Web UI listens on port **20910**.
+
+Syncthing does not require advanced DNS options. If you want to add DNS options, see [Advanced DNS Settings](#advanced-dns-settings) below.
+
+Accept the default resource limit values for CPU and memory or select **Enable Pod resource limits** to show the CPU and memory limit fields, then enter the new values you want to use for Syncthing. See [Resource Configuration Settings](#resource-configuration-settings) below for more information.
+
+Click **Install**. 
+The system opens the **Installed Applications** screen with the Syncthing app in the **Deploying** state.
+After installation completes the status changes to **Running**. 
+
+{{< trueimage src="/images/SCALE/Apps/SyncthingChartsInstalled.png" alt="Syncthing Installed" id="Syncthing Installed" >}}
```

Comment:
This image includes a few details that look like they're in the DF apps but not showing up in my EE install. Could use an update.